package.json

```diff
@@ -17,6 +17,7 @@
         "@netflix/nerror": "^1.1.0",
         "ajv": "^6.10.0",
         "ajv-errors": "^1.0.1",
+        "ajv-formats-date-comparison": "github:CriminalInjuriesCompensationAuthority/ajv-formats-date-comparison#v0.0.1",
```

Comment:
Sort of unrelated to this PR, but should the first version of this not be 0.1.0 rather than 0.0.1?